src/aliceVision/image/convertionOpenCV.hpp

```diff
@@ -81,13 +81,35 @@ inline cv::Mat imageRGBAToCvMatBGR(const image::Image<image::RGBAfColor>& img, i
  */
 inline void cvMatBGRToImageRGBA(const cv::Mat& img, image::Image<image::RGBAfColor>& imageOut)
 {
+    const float charToFloat = 1.0f / 255.0f;
     for(int row = 0; row < imageOut.Height(); row++)
```

Comment:
I think it would make sense to move the guts of this function to something like this:
```
template<class CVType, class Converter>
void cvMatBGRToImageRGBAImpl(const cv::Mat& img, image::Image<image::RGBAfColor>& imageOut, const Converter& cvt)
{
    for(int row = 0; row < imageOut.Height(); row++)
    {
        const CVType* rowPtr = img.ptr<CVType>(row);
        for(int col = 0; col < imageOut.Width(); col++)
        {
            const cv::Vec3f& matPixel = rowPtr[col];
            imageOut(row, col) = converter(imageOut(row, col), rowPtr[col]);
        }
     }
}
```

All conversion details would be put into one-line Converter `converter` lambda parameter.

Reply:
Done, although slightly differently.